httpReq.js

```diff
@@ -28,13 +30,31 @@
  *      }
  *  });
  */
-export default function httpReq(path, options = {}) {
-    /* globals dw */
-    const { payload, baseUrl, raw, ...opts } = {
+export default function httpReq(
+    path,
+    options = {},
+    csrfCookieName = 'crumb',
+    csrfTokenHeader = 'X-CSRF-Token',
+    csrfSafeMethods = new Set(['get', 'head', 'options', 'trace']) // according to RFC7231
```

Comment:
I would just hard-code this set outside the `httpReq` function. if the intention was to allow bypassing the CSRF header we should instead add an argument like `disableCsrf: false`.

Reply:
It was just a code style decision. I've changed it to constants though.